drivers/soundwire/bus.c

```diff
@@ -1293,15 +1293,15 @@ static int sdw_initialize_slave(struct sdw_slave *slave)
 		status = sdw_read_no_pm(slave, SDW_SCP_INT1);
 		if (status < 0) {
 			dev_err(&slave->dev,
-				"SDW_SCP_INT1 (PARITY) read failed:%d\n", status);
+				"SDW_SCP_INT1 read failed:%d\n", status);
```

Comment:
so now this error message is again identical to the one above at line 1277, would be interesting to know what the objection against this change was?

Reply:
not sure I follow, the code here is identical as in the first pass, the only change is literally a change from u32 to u64.

maybe I screw-up in the rebase/squashes.

Reply:
My question was basically - why do you have to revert https://github.com/thesofproject/linux/commit/528a6e2d211638fcf71ccb555f224e1f0718d281 ? If it is about patch order, and you do *partially* restore that patch later in the series as https://github.com/thesofproject/linux/pull/2753/commits/66ca3a8cee6cba5b4959b8ba902b2f65bb51f739 - but only partially. What am I missing?

Reply:
The split is now cleanups and then addition of quirks, instead of the other way around.
I may have missed some parts though, will relook later today.

Reply:
@lyakh I rechecked the patches and don't see any misses. I only see a single occurrence of messages related to SDW_SCP_INT1. Please re-take a look if you don't mind.

Reply:
@plbossart you're right, those changes disappeared from the "uniquify" patch but those log entries are now directly added with unique strings, so, it's good, thanks.